### PR TITLE
Ardupilot downstream changes

### PR DESCRIPTION
**Pull request by amilcarlucas**

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -2960,42 +2960,42 @@
       </entry>
     </enum>
     <enum name="POSITION_TARGET_TYPEMASK">
-      <description>Bitmap to indicate which dimensions should be ignored by the vehicle: a value of 0b0000000000000000 or 0b0000001000000000 indicates that none of the setpoint dimensions should be ignored. If bit 10 is set the floats afx afy afz should be interpreted as force instead of acceleration.</description>
+      <description>Bitmap to indicate which dimensions should be ignored by the vehicle: a value of 0b0000000000000000 or 0b0000001000000000 indicates that none of the setpoint dimensions should be ignored. If bit 9 is set the floats afx afy afz should be interpreted as force instead of acceleration.</description>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: What's up with "If bit 9 is set" vs "If bit 10 is set"?

File: message_definitions/v1.0/common.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
         <description>SLUGS autopilot, http://slugsuav.soe.ucsc.edu</description>
       </entry>
       <entry value="3" name="MAV_AUTOPILOT_ARDUPILOTMEGA">
-        <description>ArduPilotMega / ArduCopter, http://diydrones.com</description>
+        <description>ArduPilot - Plane/Copter/Rover/Sub/Tracker, http://ardupilot.org</description>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: Shouldn't all changes to common.xml be made upstream (here) and moved downstream (to ArduPilot)? The actual change is great.

File: message_definitions/v1.0/common.xml [hunk 3]
```diff
@@ -2960,42 +2960,42 @@
       </entry>
     </enum>
     <enum name="POSITION_TARGET_TYPEMASK">
-      <description>Bitmap to indicate which dimensions should be ignored by the vehicle: a value of 0b0000000000000000 or 0b0000001000000000 indicates that none of the setpoint dimensions should be ignored. If bit 10 is set the floats afx afy afz should be interpreted as force instead of acceleration.</description>
+      <description>Bitmap to indicate which dimensions should be ignored by the vehicle: a value of 0b0000000000000000 or 0b0000001000000000 indicates that none of the setpoint dimensions should be ignored. If bit 9 is set the floats afx afy afz should be interpreted as force instead of acceleration.</description>
       <entry value="1" name="POSITION_TARGET_TYPEMASK_X_IGNORE">
-        <description>0b0000000000000001 Ignore position x</description>
+        <description>Ignore position x</description>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: FYI very happy with this.

**Comment by hamishwillee** [comment]
So my general take on this is that for dialect changes from the dialect repo we should just accept them - after checking of course whether they might reasonably be better in Common (I assume this is not the case?)

For the common changes, generally better if they come here first and propagate down to ArduPilot. That said, they are mostly good/obvious except: https://github.com/mavlink/mavlink/pull/1008#discussion_r226500007

File: message_definitions/v1.0/common.xml (hunk 1 above)
Comment (amilcarlucas) [reply message_definitions/v1.0/common.xml]: The LSB is in 99,9% of the books numbered as bit0, so this is bit9
The original description started count at bit 1 hence it called bit 9 bit 10

**Comment by amilcarlucas** [comment]
@LorenzMeier can I rebase and merge this?

**Comment by LorenzMeier** [comment]
Missed this over the weekend - good to go!

**Pull request merged**